### DISCUSSION TITLE
How to run using Docker?

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#1694] [General]
I'm trying to get this running using Docker

I get as far as running the command: 

`docker build -t only-fans . && docker run -it --rm --name onlyfans -v ${PWD}/.settings:/usr/src/app/.settings -v ${PWD}/.profiles:/usr/src/app/.profiles -v ${PWD}/.sites:/usr/src/app/.sites only-fans`

I get the following error:

`failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount444144095/Dockerfile: no such file or directory`

I've tried everything. I have no such directory. Am I suppose to be running the command in terminal? Am I missing something?
Thanks

**Comment by shanold** [comment]
you have to git clone https://github.com/UltimaHoarder/UltimaScraper first.
